_posts/en/newsletters/2025-10-31-newsletter.md

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Bitcoin Optech Newsletter #378'
+permalink: /en/newsletters/2025/10/31/
+name: 2025-10-31-newsletter
+slug: 2025-10-31-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces four vulnerabilities affecting older versions of
+the Bitcoin Core full node. Also included are our regular sections summarizing
+popular questions and answers on the Bitcoin Stack Exchange, announcing new
+releases and release candidates, and describing notable changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Disclosure of four low severity vulnerabilities in Bitcoin Core:**
+  Antoine Poinsot recently [posted][poinsot disc] to the Bitcoin Development
+  Mailing List four Bitcoin Core security advisories for low severity
+  vulnerabilities that were fixed in [Bitcoin Core 30.0][]. According to the
+  [disclosure policy][disc pol], a low severity vulnerability is disclosed two
```

Comment:
alternative link could be our coverage of that policy in the newsletter previously